custom_components/aguaiot/const.py

```diff
@@ -92,11 +104,21 @@ class AguaIOTSensorEntityDescription(SensorEntityDescription):
     ),
     AguaIOTSensorEntityDescription(
         key="real_power_get",
-        name="Real Power",
+        name="Real Pellet Power",
```

Comment:
I would propose to not change this sensor name, it might confuse people already using the plugin. Also it looks a bit odd when you have a non-hybrid stove (because all the other controls do not specifically mention "Pellet")

Reply:
I was wondering if you'll ask for it.
No problem, i'll do it.